### PR TITLE
fix(release): fix broken macos debug build

### PR DESCRIPTION
**Pull request by daniellacosse**
note the [full logs](https://github.com/Jigsaw-Code/outline-client/runs/5858505229?check_suite_focus=true#step:6:1711) in the macos debug build ci (before cordova would stop [here](https://github.com/Jigsaw-Code/outline-client/runs/5858460927?check_suite_focus=true#step:6:780) and throw the wrong status code)

### CODE REVIEW

File: cordova-plugin-outline/plugin.xml [hunk 1]
```diff
@@ -117,8 +117,8 @@
     <source-file src="apple/src/OutlineSentryLogger.swift" />
     <source-file src="apple/src/OutlineTunnel.swift" />
     <source-file src="apple/src/OutlineVpn.swift" />
+    <resource-file src="apple" />
```
Comment (daniellacosse) [inline cordova-plugin-outline/plugin.xml:120]: Alright so, I really dislike this. Basically what this does is copies the whole of `cordova-plugin-outline/apple` into `platforms/osx/Outline/Resources`, where before, it was copying the contents of the `vpn` folder straight into that folder. That's incorrect - we need it to copy it to `platforms/osx/Outline/Resources/vpn` instead.

[Allegedly the `target` property is supposed to set where the resource is copied _to_.](https://cordova.apache.org/docs/en/latest/plugin_ref/spec.html#resource-file) but no matter what I set it to, it did not affect the copy location, implying this option is completely ignored in `cordova-osx`. So effectively what we have here is very brittle - if I switch the order of the `resource-file` tags, it breaks, and if I try to be more specific and only copy what we need, it breaks.

However, for now this satisfies the requirement of having the files where we need them in the prepared project and OSX builds. The alternative is to keep it as it was and then have additional steps in the script to move things around - but that's also just as brittle, really.

Comment (fortuna) [reply cordova-plugin-outline/plugin.xml:120]: I don't understand. Shouldn't that be `apple/vpn`? And why does changing the order help?

Comment (daniellacosse) [reply cordova-plugin-outline/plugin.xml:120]: Let's go over it once you're back in the office.

Comment (fortuna) [reply cordova-plugin-outline/plugin.xml:120]: Wait, HEAD seems to be working in my machine. I see this directory structure:
image

Notice that the Resources look fine and the Outline.app is there after the build using `npm run action gulp build osx`.

I was able to successfully run the app: 
image

Maybe it's an issue in your environment?

Comment (fortuna) [reply cordova-plugin-outline/plugin.xml:120]: I am, however, seeing an error on release builds with `SENTRY_DSN=https://foo.ba npm run action gulp build osx -- --buildMode=release`:

```
error: VpnExtension has conflicting provisioning settings. VpnExtension is automatically signed, but code signing identity Mac Distribution has been manually specified. Set the code signing identity value to "Apple Development" in the build settings editor, or switch to manual signing in the Signing & Capabilities editor. (in target 'VpnExtension' from project 'Outline')
error: Outline has conflicting provisioning settings. Outline is automatically signed, but code signing identity Mac Distribution has been manually specified. Set the code signing identity value to "Apple Development" in the build settings editor, or switch to manual signing in the Signing & Capabilities editor. (in target 'Outline' from project 'Outline')
** BUILD FAILED **
```

Comment (fortuna) [reply cordova-plugin-outline/plugin.xml:120]: I wonder if the CI errors are due to architecture issues. My machine is intel.

Comment (daniellacosse) [reply cordova-plugin-outline/plugin.xml:120]: Yeah the signing thing I'm working on now.

What version of `cordova-osx` do you have locally?

Comment (fortuna) [reply cordova-plugin-outline/plugin.xml:120]: package-lock.json says `git+ssh://git@github.com/Jigsaw-Code/cordova-osx.git#07e62a53aa6a8a828fd988bc9e884c38c3495a67`

The package.json in the installed package says `"version": "7.0.0-dev"`, which doesn't match the current head: https://github.com/apache/cordova-osx/blob/16b56e4a9297c5195a06a57450aeb983e7053184/package.json#L3

Comment (daniellacosse) [reply cordova-plugin-outline/plugin.xml:120]: Yep, you're package lock is different, the version on master is "git+ssh://git@github.com/apache/cordova-osx.git#9b1480d20d6626c0f5a32294e002b87a0cb35c0a"

Comment (daniellacosse) [reply cordova-plugin-outline/plugin.xml:120]: Yep, setting it explicitly to `07e62a53aa6a8a828fd988bc9e884c38c3495a67` works. So, as I suspected, the file resolution broke between 7.0.0 and 7.1.0

Comment (daniellacosse) [reply cordova-plugin-outline/plugin.xml]: Oh wow, this breaks windows

Comment (daniellacosse) [reply cordova-plugin-outline/plugin.xml]: Unfortunately, setting it to exactly `7.0.0` doesn't work. We're stuck.

**Comment by daniellacosse** [comment]
closing in favor of https://github.com/Jigsaw-Code/outline-client/pulls#:~:text=Sort-,Fix%20macOS,-%231251%20opened%2019

**Pull request closed without merging**